source-academy/cadet-frontend
# [WIP] Implement Keystroke Recordings for Assessments

### Description

Implements Simple Keystroke Recording for the `AssesmentWorkspace`

This is done using local state management within the `AssessmentWorkplace` of Source Academy 

Fixes # (issue)

### Type of change


### How to Test

Making Changes within the Editor and Clicking Save should allow a payload to be logged into your console. 
TODO: Add handlers to send payload to backend. 

### Checklist

Please delete options that are not relevant.
